magefiles/magefile.go

```diff
@@ -823,8 +824,9 @@ func GenerateTestSuiteFile(packageName string) error {
 	return nil
 }
 
-// Remove all webhooks which with 1 day lifetime. By default will delete webooks from redhat-appstudio-qe
-func CleanWebHooks() error {
+// Remove all webhooks older than 1 day from GitHub repo.
+// By default will delete webooks from redhat-appstudio-qe
```

Comment:
```suggestion
// By default will delete webhooks from redhat-appstudio-qe
```
![harold](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS15Cb4eyDctl2Y1N8OB7vmYNEWOGU7NiGlMw&s)

Reply:
Updated!